.github/workflows/docker-dev.yml

```diff
@@ -0,0 +1,73 @@
+name: Docker - Development Build
+
+#Run this action against any commit on main.
```

Comment:
"against any commit in main" is great insofar as it goes, but I also want to automatically create a new Dev build whenever a commit is pushed to any of the (currently 6) integrated linter projects.  Any thoughts on how to do that via GitHub Actions?

Reply:
We could combine your "against any commit in main" approach with something else (external to GitHub Actions if needed) that regularly checks the linter repositories and stores the latest commit hashes in a file in this repository.  Whenever that file changes (and is automatically committed and pushed by the "something else"), the "against any commit in main" Action would kick in.

I'm hoping there's a better solution though.  (I'd prefer to avoid "polluting" this repository with zillions of commits just to track the commits in the linter repos).

Reply:
I think we can do this, but potentially should use a different workflow for that purpose. Potentially a different workflow file for each included linter. 

I'd recommend we make that an issue to be done separately

Reply:
Yeah OK.  I'll merge this PR as is, then we'll go from there.  Thanks again!